src/Features/LanguageServer/Protocol/Extensions/ProtocolConversions.cs

```diff
@@ -167,7 +167,7 @@ public static LSP.DiagnosticSeverity DiagnosticSeverityToLspDiagnositcSeverity(D
                 case DiagnosticSeverity.Hidden:
                     return LSP.DiagnosticSeverity.Hint;
                 case DiagnosticSeverity.Info:
-                    return LSP.DiagnosticSeverity.Information;
+                    return LSP.DiagnosticSeverity.Hint;
```

Comment:
Maybe add a comment as to why we are returning `Hint` here and link to an issue tracking this being reverted.

Reply:
I don't think we will necessarily be reverting it.  This is just how the LSP client handles squiggles - info is purple and Hint is two dots.

Reply:
Hidden should also not be hint, but it will be until we have support for hidden diagnostics - https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1063158

Reply:
Interesting choice.